Share with Gifski/loadingHtml.html

```diff
@@ -0,0 +1,61 @@
+<html>
```

Comment:
I would prefer a more Mac-like spinner like this one: https://codepen.io/jmak/pen/LsCet

Reply:
Actually, why do you even need a web spinner? You can just overlay a native view over the webview with a native spinner.